tools/tpm2_import.c

```diff
@@ -51,6 +51,7 @@
 #include "tpm_kdfa.h"
 #include "tpm2_options.h"
 #include "tpm2_util.h"
+#include "tpm2_tool.h"
```

Comment:
Can you please keep the headers sorted alphabetically? So this should be after tpm2_options.h.